pkg/proxy/winkernel/proxier.go

```diff
@@ -989,7 +989,7 @@ func (proxier *Proxier) syncProxyRules() {
 	prevNetworkID := proxier.network.id
 	updatedNetwork, err := hns.getNetworkByName(hnsNetworkName)
 	if updatedNetwork == nil || updatedNetwork.id != prevNetworkID || isNetworkNotFoundError(err) {
-		klog.InfoS("The HNS network %s is not present or has changed since the last sync. Please check the CNI deployment", "hnsNetworkName", hnsNetworkName)
+		klog.InfoS("The HNS network is not present or has changed since the last sync. Please check the CNI deployment", "hnsNetworkName", hnsNetworkName)
```

Comment:
please change it to
"The HNS network is not present or has changed since the last sync, please check the CNI deployment"

Reply:
You may want to change it in the same fashion for the complete file, thanks!